main/reference/zoe-api/user-seat.md

```diff
@@ -14,6 +14,13 @@ and, if it's allowed for this seat, call **tryExit()**.
 
 Take care when sharing a **UserSeat**, since it includes authority to (attempt to) exit the seat.
 
+These methods are all documented as returning Promises, since they are always called remotely,
+but that obscures the fact that they are expected to resolve at semantically different times.
+Some return a value immediately, some do a small amount of computation before returning, and
```

Comment:
```suggestion
Some return a value promptly, some do a small amount of computation before returning, and
```

You use "promptly" below, and for this purpose it is much clearer than immediately. Do we define either term anywhere? I like Norm's definition of "prompt" or "promptly", that it will happen without any further actions needed by any other party. Also, I would say, implies only computational delays such as round trips, rather than scheduled timing delays.

Thus, unclear if you meant "do a small amount of computation before returning" to be a separate category. From the pov of a remote caller, I'd expect these to also be prompt and to not be in a different category from your "immediately".

I would like to reserve terms like "immediate" and "immediately" for things that happen synchronously in the same turn, which is never true for remote calls.